clang/test/dpct/lit.local.cfg

```diff
@@ -5,6 +5,54 @@ import sys
 import platform
 import subprocess
 import shutil
+header_files = []
```

Comment:
add comment for new added functions on purpose "find out some test cases, and disable them in case header files are not available".